app/src/main/java/org/simple/clinic/bp/entry/BloodPressureEntryEffect.kt

```diff
@@ -7,3 +7,7 @@ object PrefillDateForNewEntry : BloodPressureEntryEffect()
 object HideBpErrorMessage : BloodPressureEntryEffect()
 
 object ChangeFocusToDiastolic : BloodPressureEntryEffect()
+
+object ChangeFocusToSystolic : BloodPressureEntryEffect()
+
+data class SetSystolic(val systolic: String) : BloodPressureEntryEffect()
```

Comment:
I am assuming that the `systolic` property will eventually become a part of the model.